mmv1/products/vertexai/FeatureOnlineStore.yaml

```diff
@@ -0,0 +1,225 @@
+  - !ruby/object:Api::Type::String
+    name: 'updateTime'
+    output: true
+    description: |
+      The timestamp of when the featurestore was last updated in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits.
+  - !ruby/object:Api::Type::KeyValueLabels
+    name: 'labels'
+    description: |
+      A set of key/value label pairs to assign to this Featurestore.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'dedicatedServingEndpoint'
+    description: |
+       The dedicated serving endpoint for this FeatureOnlineStore, which is different from common vertex service endpoint. Only need to set when you choose Optimized storage type or enable EmbeddingManagement. Will use public endpoint by default.
+    min_version: beta
+    default_from_api: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'publicEndpointDomainName'
+        description: |
+          populated with the domain name to use for this FeatureOnlineStore
+        output: true
+      - !ruby/object:Api::Type::String
+        name: 'serviceAttachment'
+        description: |
+          name of the service attachment resource. Populated if private service connect is enabled and after FeatureViewSync is created.
+        output: true
+      - !ruby/object:Api::Type::NestedObject
+        name: 'privateServiceConnectConfig'
+        description: |
+          Private service connect config.
+        default_from_api: true
+        properties:
+          - !ruby/object:Api::Type::Boolean
+            name: 'enablePrivateServiceConnect'
+            required: true
+            default_from_api: true
+            description: |
+              If set to true, customers will use private service connection to send request. Otherwise, the connection will set to public endpoint.
+          - !ruby/object:Api::Type::Array
+            name: 'projectAllowlist'
+            default_from_api: true
+            description: |
+              A list of Projects from which the forwarding rule will target the service attachment.
+            item_type: Api::Type::String
+  - !ruby/object:Api::Type::NestedObject
+    name: 'embeddingManagement'
+    description: |
+      The settings for embedding management in FeatureOnlineStore. Embedding management can only be used with BigTable.
+    min_version: beta
+    properties:
+      - !ruby/object:Api::Type::Boolean
+        name: 'enabled'
+        description: |
+          Enable embedding management.
+        immutable: true
+  - !ruby/object:Api::Type::NestedObject
+    name: 'bigtable'
+    description: |
+      Contains settings for the Cloud Bigtable instance that will be created to serve featureValues for all FeatureViews under this FeatureOnlineStore.
+    conflicts:
+      - optimized
+    exactly_one_of:
+      - bigtable
+      - optimized
```

Comment:
The properties `conflicts` and `exactly_one_of` are very similar so we can remove one of them here (and in the description of the optimized field). When deciding which one to remove we need to decide whether it's valid for users to have both bigtable and optimized unset in their configuration.

When I created the resource with neither field set I got this error:

```
  "error": {
    "code": 400,
    "message": "FeatureOnlineStore must have either Bigtable or Optimized storage enabled.",
    "status": "INVALID_ARGUMENT"
  }
```

So I think we should keep `exactly_one_of` and remove `conflicts` from here

Reply:
Done